src/patternfly/components/FormControl/form-control.hbs

```diff
@@ -2,4 +2,4 @@
   {{#if form-control--attribute}}
     {{{form-control--attribute}}}
 	{{/if}}
-{{#if input}}>{{else}}>{{> @partial-block}}</{{controlType}}>{{/if}}
+{{#if input}}{{#if form-control--value}} value="{{form-control--value}}"{{/if}}>{{else}}>{{> @partial-block}}</{{controlType}}>{{/if}}
```

Comment:
You shouldn't need this, you can pass `value="foo"` to `form-control--attribute`

Here are some examples

```
src/patternfly//components/Pagination/examples/Pagination.md:      {{> form-control input="true" controlType="input" form-control--attribute='aria-label="Current page" type="number" min="1" max="4" value="1"'}}
src/patternfly//components/Pagination/examples/Pagination.md:      {{> form-control input="true" controlType="input" form-control--attribute='aria-label="Current page" type="number" min="1" max="4" value="1"'}}
src/patternfly//components/Pagination/examples/Pagination.md:      {{> form-control input="true" controlType="input" form-control--attribute='aria-label="Current page" type="number" min="1" max="4" value="1"'}}
src/patternfly//components/ClipboardCopy/examples/ClipboardCopy.md:    {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'type="text" value="This is editable" id="text-input-' clipboard-copy--id '" aria-label="Copyable input"')}}
src/patternfly//components/ClipboardCopy/examples/ClipboardCopy.md:    {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'readonly type="text" value="This is read-only" id="text-input-' clipboard-copy--id '" aria-label="Copyable input"')}}
src/patternfly//components/ClipboardCopy/examples/ClipboardCopy.md:    {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'type="text" value="This is an editable version of the Copy to Clipboard Component that has an expandable section. Got a lot of text here, need to see all of it? Click that arrow on the left side and check out the resulting expansion." id="text-input-' clipboard-copy--id '" aria-label="Copyable input"')}}
```

Reply:
great, thanks.